pages/api/_components/template/programme.js

```diff
@@ -123,20 +121,18 @@ export default async function getProgrammeHtml(data) {
         <div class="row pl-4 ml-1">
         ${
           data.tags !== null &&
-          data.tags.map(
-            (tag, index) => `
-            <div class="p-1">
-            <img
-              src="https://raw.githubusercontent.com/codinasion/codinasion/master/image/language/${
+          data.tags.map((tag, index) => (
+            <div key={index} class='p-1'>
+              <img
+                src={`https://raw.githubusercontent.com/codinasion/codinasion/master/image/language/${
                 FormatTag(tag).tag
-              }.png"
-              class="rounded-circle text-center"
-              alt="image"
-              style="width: 40px; max-height: 40px"
-            />
-          </div>
-        `
-          )
+              }.png`}
+                class='rounded-circle text-center'
+                alt='image'
+                style='width: 40px; max-height: 40px'
+              />
+            </div>
+          ))
         }
```

Comment:
Hey @smdpro 

Sorry, but this will not fix the issue

### Reason

- `getProgrammeHtml` return html as string, but the updated code will return `string` + `component` + `string`. It should be all string

![Error !!!](https://user-images.githubusercontent.com/54644599/177242262-d59cfb1a-46e2-4f91-ad9f-24718e21bcd3.png)

Reply:
Hi @harshraj8843
I checked it locally, and fetching data has not returned JSON format. 
for route :
**http://localhost:3000/api?url=https://codinasion.vercel.app/programme/add-two-numbers**
console output :
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
info  - Loaded env from /home/smd/codes/openSource/codinasion-og/.env
FetchError: invalid json response body at https://raw.githubusercontent.com/codinasion/codinasion-data/master/data/contributors.json reason: Unexpected token : in JSON at position 3
    at /home/smd/codes/openSource/codinasion-og/node_modules/next/dist/compiled/node-fetch/index.js:1:49606
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async handler (/home/smd/codes/openSource/codinasion-og/.next/server/pages/api.js:573:30)
    at async Object.apiResolver (/home/smd/codes/openSource/codinasion-og/node_modules/next/dist/server/api-utils/node.js:185:9)
    at async NextNodeServer.runApi (/home/smd/codes/openSource/codinasion-og/node_modules/next/dist/server/next-server.js:395:9)
    at async Object.fn (/home/smd/codes/openSource/codinasion-og/node_modules/next/dist/server/base-server.js:496:37)
    at async Router.execute (/home/smd/codes/openSource/codinasion-og/node_modules/next/dist/server/router.js:226:36)
    at async NextNodeServer.run (/home/smd/codes/openSource/codinasion-og/node_modules/next/dist/server/base-server.js:606:29)
    at async NextNodeServer.handleRequest (/home/smd/codes/openSource/codinasion-og/node_modules/next/dist/server/base-server.js:321:20) {
  type: 'invalid-json'
}
There is something wrong with calling API
call  [ https://raw.githubusercontent.com/codinasion/codinasion-data/master/data/contributors.json] in your browser. You will get 404: Not Found

Reply:
> Hi @harshraj8843 I checked it locally, and fetching data has not returned JSON format.

Hey @smdpro, sorry for the late reply.

This issue occurred because `codinasion-data` (the repo from which json data is being fetched) is `private`. So, it can only be accessed by Codinasion members.

[Become a member of codinasion](https://github.com/codinasion/.github/issues/new?assignees=&labels=welcome+%F0%9F%8E%89%F0%9F%8E%89%F0%9F%8E%89&template=invitation.yml&title=Please+invite+me+to+Codinasion)

And please don't forget to add your `personal access token` in `.env` (GITHUB_TOKEN=your_personal_access_token)

> "codinasion" in fetch URL is not a dynamic variable, just a static string?

Yes, but this is intentional for the purpose of review. It will be updated later